src/codegens/golang.ts

```diff
@@ -133,9 +135,49 @@ export default class Golang extends CodeGen {
       return [...typings, `\t${title.padEnd(titleMaxLength)} *${title}`];
     }, []);
 
+    const tit = this.getSafeTitle(s.title as string);
+
+    // Assuming that AnyOfs are either an {}, or an [];
```

Comment:
![image](https://user-images.githubusercontent.com/1787231/79598506-f01c6b00-8098-11ea-8b19-e412a8e3fa59.png)
https://json-schema.org/understanding-json-schema/reference/combining.html

AFAIK anyOf must be an array